# HG changeset patch
# User sbr <sebastien@brissaud.name>
# Date 1375221816 -7200
#      Wed Jul 31 00:03:36 2013 +0200
# Node ID 806a382637a32a8b11b722956320b168524aaa53
# Parent  f24a577343d33afa96bb2b3677e7eef3125ec889
-Fix (r20055): '+' isn't a valid scancode and shouldn't be used as hotkey

diff --git a/src/hotkeys.cpp b/src/hotkeys.cpp
--- a/src/hotkeys.cpp
+++ b/src/hotkeys.cpp
@@ -54,7 +54,6 @@ static const KeycodeNames _keycode_to_na
 	{"F11", WKC_F11},
 	{"F12", WKC_F12},
 	{"PAUSE", WKC_PAUSE},
-	{"PLUS", (WindowKeyCodes)'+'},
 	{"COMMA", WKC_COMMA},
 	{"NUM_PLUS", WKC_NUM_PLUS},
 	{"NUM_MINUS", WKC_NUM_MINUS},
